llvm/lib/DWARFLinker/DWARFStreamer.cpp

```diff
@@ -874,6 +880,10 @@ void DwarfStreamer::emitLineTablePrologueV5IncludeAndFileTable(
 
     LineSectionSize += MS->emitULEB128IntValue(dwarf::DW_LNCT_directory_index);
     LineSectionSize += MS->emitULEB128IntValue(dwarf::DW_FORM_data1);
+    if (InlineSources) {
+      LineSectionSize += MS->emitULEB128IntValue(dwarf::DW_LNCT_LLVM_source);
+      LineSectionSize += MS->emitULEB128IntValue(P.FileNames[0].Name.getForm());
```

Comment:
Might be worth storing `P.FileNames[0].Name.getForm()` in a temporary and sharing it between line 879 and here.